include/ck/utility/amd_smfmac.hpp

```diff
@@ -5,6 +5,10 @@
 #pragma once
 
 namespace ck {
+// Define the common macro for gfx94x models
+#if defined(__gfx940__) || defined(__gfx941__) || defined(__gfx942__)
+#define __gfx94__
+#endif
```

Comment:
Why we need this macro here? It's duplicated from https://github.com/ROCm/composable_kernel/blob/497ccb872b6a9a921c01df4dca49dac7cb242c72/include/ck/ck.hpp#L59-L60 which is included at the beginning of this file.